Docs/developer_guide/build_instructions/overview.md

```diff
@@ -38,3 +38,16 @@ Customized editions of Slicer can be generated without changing Slicer source co
 - `Slicer_EXTENSION_SOURCE_DIRS`: Defines additional extensions that will be included in the application package as built-in modules. Full paths of extension source directories has to be specified, separated by semicolons.
 
 More customization is available by using [SlicerCustomAppTemplate](https://github.com/KitwareMedical/SlicerCustomAppTemplate) project maintained by Kitware.
+
+## Runtime environment variables
```

Comment:
Thanks for adding the documentation.

Reply:
Yes, now that documentation is tightly coupled with the source code, it is feasible to describe and maintain such details in the documentation. All the work invested in the infrastructure is paying off.

Reply:
Agreed. Definitely the way to go